---
layout: project
permalink: /:title/
category: projects

meta:
  keywords: "maize, transposons, transposable elements, next-generation sequencing, linkage disequilibrium"

project:
  title: "Transposons in a maize population"
  paper: "Whole genome variation of transposable element insertions in a maize diversity panel"
  authors: "Qiu Y, Oâ€™Connor CH, <a href='https://rdcoletta.github.io/'>Della Coletta R</a>, et al."
  type: "Collaborative project"
  doi: "10.1093/g3journal/jkab238"
  url: "https://doi.org/10.1093/g3journal/jkab238"
  logo: "/assets/images/projects/te-pop/logo.jpeg"
  logo-cc: "credits: Cold Spring Harbor Laboratory Library & Archives"
  tech: "UNIX, TASSEL, Plink, R, bedtools, vcftools"

journal:
  title: "G3: Genes, Genomes, Genetics"
  url: "https://academic.oup.com/g3journal"
  impact: "3.154"
  year: "2021"

images:
  - image:
    url: "/assets/images/projects/te-pop/ld1.png"
    alt: ""
    desc: "Linkage disequilibrium (LD) distribution between TEs and SNPs with highest LD to a TE"
  - image:
    url: "/assets/images/projects/te-pop/ld2.png"
    alt: ""
    desc: "LD between TEs and SNPs per TE category"
---
<p>Transposable elements (TEs), or "jumping genes", are hard to study due to their repetitive nature and abundance in crop genomes. However, when using short-read sequencing data to characterize them in a population, things get even more complicated. In this study, we implemented a machine learning method to classify the presence and absence of TEs in a diversity panel with 500+ maize lines. By calculating linkage disequilibrium (LD) between TEs and single nucleotide polymorphisms (SNPs), I observed that nearly 20% of the TEs were not being tagged by an SNP. This result potentially represents information that has not been well captured in previous SNP-based marker-trait association studies. The step-by-step of the LD analysis can be found on this <a href='https://github.com/HirschLabUMN/TE_variation/tree/master/TE-SNP_LD_methods'>Github page</a>.</p>
