---
layout: project
permalink: /:title/
category: projects

meta:
  keywords: "nonhost resistance, brachypodium, wheat stem rust, bulked segregant analysis"

project:
  title: "Wheat stem rust susceptibility genes"
  paper: "Identification of candidate susceptibility genes to <em>Puccinia graminis</em> f. sp. <em>tritici</em> in wheat"
  authors: "Henningsen EC, Omidvar V, <a href='https://rdcoletta.github.io/'>Della Coletta R</a> et al."
  type: "Collaborative project"
  doi: "10.3389/fpls.2021.657796"
  url: "https://doi.org/10.3389/fpls.2021.657796"
  logo: "/assets/images/projects/wsr/logo.png"
  logo-cc: "credits: Martin Grandjean, CC BY-SA 4.0, via Wikimedia Commons"
  tech: "R, python, UNIX"

journal:
  title: "Frontiers in Plant Science"
  url: "https://www.frontiersin.org/journals/plant-science"
  impact: "4.407"
  year: "2021"

images:
  - image:
    url: "/assets/images/projects/wsr/networks.png"
    alt: ""
    desc: "Networks containing orthologs of DMR6, a candidate susceptibility gene, in a susceptible wheat variety, resistant wheat variety and Brachypodium distachion"
---
<p> During my Master's, I analyzed genomic data from <em>Brachypodium distachyon</em> to identify regions associated with non-host resistance to wheat stem rust. For this collaborative project, I built gene co-expression networks from wheat and Brachypodium transcriptomic data to link candidate susceptibility genes (identified by differential expression analysis) with their biological pathways. This approach allowed us to identify potential targets for future gene editing studies. Removing or modifying these putative susceptibility genes could result in a new way to generate wheat plants resistant to stem rust. The step-by-step of the network analysis can be found on this <a href='https://github.com/rdcoletta/stem-rust_networks'>Github page</a>.</p>
